gke-scripts/50-td-components.sh

```diff
@@ -0,0 +1,64 @@
+#! /bin/bash
+
+function create_health_check {
+  if [ $# -lt 2 ]
+  then
+    echo "usage: create_health_check <health_check_name> <health_check_port>"
+    exit 1
+  fi
+
+  gcloud compute health-checks create grpc $1 --enable-logging --port $2
+}
+
+function create_backend_service {
+  if [ $# -lt 3 ]
+  then
+    echo "usage: create_backend_service <backend_service_name> <neg_name>"
+    exit 1
+  fi
+
+  local BACKEND_SERVICE_NAME="$1"
+  local HEALTH_CHECK_NAME="$2"
+  local NEG_NAME="$3"
+
+  gcloud compute backend-services create ${BACKEND_SERVICE_NAME} \
+    --global \
+    --load-balancing-scheme=INTERNAL_SELF_MANAGED \
+    --protocol=GRPC \
+    --health-checks ${HEALTH_CHECK_NAME}
```

Comment:
nit: Fix argument style to be consistent everywhere.  Sometimes it's `--flag VAL`, sometimes `--flag=VAL`. I recommend `--flag=VAL`.